graphql/src/api/resolvers/system-properties/system-properties.js

```diff
@@ -111,21 +111,21 @@ const typeDefs = `
   }
 `
 
-const getBuildInfo = () => {
-  /* eslint-disable */
-  const commitHash = __COMMIT_HASH__
-  const isDirty = __IS_DIRTY__
-  const commitDate = __COMMIT_DATE__
-  /* eslint-enable */
+// const getBuildInfo = () => {
```

Comment:
do we want these comments around?

Reply:
We changed ours to 

```ts
let commitHash = 'no-hash'
let commitDate = new Date()
let isDirty = true
try {
  commitHash = exec('git rev-parse --short HEAD').toString()
  commitDate = exec('git log -1 --pretty=format:%cI').toString()
  isDirty =
    exec('git status')
      .toString()
      .indexOf('working directory clean') === -1
} catch (e) {
  console.log('Unable to get build info with git.')
}

const getBuildInfo = () => {
  return {
    commitHash,
    isDirty,
    commitDate,
    identifier: `${commitHash.trim()}${isDirty ? ' with Changes' : ''}`,
    releaseDate: commitDate,
  }
}
```

in OCTOCX but have a ticket to calculate all of them at build time so that they can go back to environment variables.